failures:

---- cfg::cfg_looks_at_rustflags_for_target stdout ----
        running `/home/build/aports/community/rust/src/rustc-1.27.0-src/build/x86_64-alpine-linux-musl/stage2-tools/x86_64-alpine-linux-musl/release/cargo build --target x86_64-unkn
own-linux-gnu`
thread 'cfg::cfg_looks_at_rustflags_for_target' panicked at '
Expected: execs
    but: exited with exit code: 101
--- stdout

--- stderr
   Compiling b v0.0.1 (file:///home/build/aports/community/rust/src/rustc-1.27.0-src/build/x86_64-alpine-linux-musl/stage2-tools/x86_64-alpine-linux-musl/cit/t356/foo/b)
error[E0463]: can't find crate for `std`
  |
  = note: the `x86_64-unknown-linux-gnu` target may not be installed

--- a/src/tools/cargo/tests/testsuite/cfg.rs
+++ b/src/tools/cargo/tests/testsuite/cfg.rs
@@ -449,7 +449,7 @@ fn cfg_looks_at_rustflags_for_target() {
         .build();
 
     assert_that(
-        p.cargo("build --target x86_64-unknown-linux-gnu")
+        p.cargo("build --target x86_64-alpine-linux-musl")
             .env("RUSTFLAGS", "--cfg with_b"),
         execs().with_status(0),
     );
--- a/src/tools/cargo/tests/testsuite/rustdoc.rs
+++ b/src/tools/cargo/tests/testsuite/rustdoc.rs
@@ -269,13 +269,13 @@
         .build();
 
     assert_that(
-        p.cargo("rustdoc --verbose --target x86_64-unknown-linux-gnu"),
+        p.cargo("rustdoc --verbose --target x86_64-alpine-linux-musl"),
         execs().with_status(0).with_stderr("\
 [DOCUMENTING] a v0.0.1 ([..])
 [RUNNING] `rustdoc --crate-name a src[/]lib.rs \
-    --target x86_64-unknown-linux-gnu \
-    -o [..]foo[/]target[/]x86_64-unknown-linux-gnu[/]doc \
-    -L dependency=[..]foo[/]target[/]x86_64-unknown-linux-gnu[/]debug[/]deps \
+    --target x86_64-alpine-linux-musl \
+    -o [..]foo[/]target[/]x86_64-alpine-linux-musl[/]doc \
+    -L dependency=[..]foo[/]target[/]x86_64-alpine-linux-musl[/]debug[/]deps \
     -L dependency=[..]foo[/]target[/]debug[/]deps`
 [FINISHED] dev [unoptimized + debuginfo] target(s) in [..]"),
     );
